### PR TITLE
Add support for RubyMotion in the gem :iphone:

### PR DESCRIPTION
**Pull request by markrickert**

### CODE REVIEW

File: lib/uk_postcode.rb [hunk 1]
```diff
@@ -108,5 +108,3 @@ def digits(s)
     s && s.tr("IO", "10")
   end
 end
-
-require "uk_postcode/version"
```
Comment (threedaymonk) [inline lib/uk_postcode.rb]: Why do you want to remove this?

**Comment by threedaymonk** [comment]
I'm very close to releasing version 2.0 using the code in the `next` branch, so you should probably look at applying your changes there.

However, this gives me something of a problem: I don't understand the reasons behind the changes you've made, and RubyMotion is a proprietary product running on a proprietary operating system running on proprietary hardware. Even if this works today, future changes I make might break it, and I'll have no way of knowing.

File: lib/uk_postcode.rb (hunk 1 above)
Comment (markrickert) [inline lib/uk_postcode.rb]: RubyMotion doesn't allow requiring in source files at runtime, Files all have to be included in the Rakefile at compile time.

**Comment by markrickert** [comment]
Awesome, I hadn't seen that other branch. I'll check that out and see if I can apply these changes to the new version.

I was looking for a way to validate and parse UK postal codes in RubyMotion and your library looked like the easiest and least complicated to implement and adding the RubyMotion support was trivial, so i figured I'd send you a PR instead of just dropping the code into my project. :)

**Comment by markrickert** [comment]
Thanks for the response. I'm moving in another direction since I only need validation and not the robust system you've created here. This looks like a great gem, but it's got a bit too much power than what I currently need :)

Cheers!
-Mark

**Pull request closed without merging**